Nio/Settings/IdentityServerSettings.swift

```diff
@@ -0,0 +1,243 @@
+import SwiftUI
+import MatrixSDK
+
+import NioKit
+
+struct IdentityServerSettingsContainerView: View {
+    @EnvironmentObject var store: AccountStore
+
+    var body: some View {
+        IdentityServerSettingsView()
+    }
+}
+
+struct ButtonModifier: ViewModifier {
+    @AppStorage("accentColor") private var accentColor: Color = .purple
+
+    func body(content: Content) -> some View {
+        content
+            .foregroundColor(.white)
+            .font(.headline)
+            .padding()
+            .frame(minWidth: 0, maxWidth: .infinity, alignment: .center)
+            .background(RoundedRectangle(cornerRadius: 15, style: .continuous)
+                            .fill(accentColor))
+            .padding(.bottom)
+    }
+}
+
+extension View {
+    func customButton() -> ModifiedContent<Self, ButtonModifier> {
```

Comment:
```suggestion
    func customButton() -> some View {
```

You should be able to simplify this a bit 😊